ext/truth/java/androidx/test/ext/truth/os/SparseBooleanArraySubject.java

```diff
@@ -0,0 +1,118 @@
+package androidx.test.ext.truth.os;
+
+import android.util.SparseBooleanArray;
+
+import com.google.common.truth.ExpectFailure;
+import com.google.common.truth.Fact;
+import com.google.common.truth.FailureMetadata;
+import com.google.common.truth.Subject;
+import com.google.common.truth.Truth;
+
+/**
+ * Subject for making assertions about {@link SparseBooleanArray}s.
+ */
+public final class SparseBooleanArraySubject extends Subject {
+
+    private final SparseBooleanArray actual;
+
+    private static Factory<SparseBooleanArraySubject, SparseBooleanArray> sparseBooleanArrays() {
+        return SparseBooleanArraySubject::new;
+    }
+
+    SparseBooleanArraySubject(FailureMetadata failureMetadata, SparseBooleanArray subject) {
+        super(failureMetadata, subject);
+        this.actual = subject;
+    }
+
+    /**
+     * Assert that a {@link SparseBooleanArray} for the given key returns the value true
+     */
+    public SparseBooleanArraySubject keyIsTrue(int key) {
```

Comment:
The key is not true, the associated value is... I think this needs a different name.

Reply:
Valid point. How about `keyReturnsTrue`?

Reply:
A function returns a value, key is data. How about `valueIsTrueAt(key)` to follow the naming conventions of SparseBooleanArray?

Reply:
Considering https://truth.dev/extension.html, maybe `hasTrueValueAt` is even better.

Reply:
> I must admit I liked the original naming best (`keyIsTrue`) although I understand that it could be misunderstood. I'll change it to `isTrueAt` and `isFalseAt` as I tend to prefer shorter function-names.

Discussion continus here: https://github.com/android/android-test/pull/884#discussion_r577130599